README.md

```diff
@@ -108,18 +108,18 @@ pop build parachain --release
 You can spawn a local network using [zombienet](https://github.com/paritytech/zombienet-sdk) as follows:
 
 ```shell
-pop up parachain -f ./tests/zombienet.toml -p https://github.com/r0gue-io/pop-node
```

Comment:
This example can still be useful for showing developers how to specify the URL of the parachain. What do you think, @brunopgalvao ?

Reply:
Restored.